.github/workflows/main.yml

```diff
@@ -45,6 +45,7 @@ jobs:
       CARGO_PROFILE_TEST_DEBUG: 1
       CARGO_INCREMENTAL: 0
       CARGO_PUBLIC_NETWORK_TESTS: 1
+      CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
```

Comment:
Should we move this to an `env` block at the top of the file so it applies to all jobs and not just this tests job?